gapic/src/main/com/google/gapid/perfetto/models/CounterInfo.java

```diff
@@ -23,10 +23,11 @@
 
 public class CounterInfo {
   private static final String LIST_SQL =
-      "select counter_id, name, ref, ref_type, description, count(value)," +
-      " min(value), max(value), avg(value) " +
-      "from counter_definitions left join counter_values using (counter_id) " +
-      "group by counter_id";
+      "select cd.counter_id, cd.name, cd.ref, cd.ref_type, cd.description, "
+      + "count(cv.value), min(cv.value), max(cv.value), avg(cv.value), p.pid "
+      + "from counter_definitions cd left join counter_values cv "
+      + "on (cd.counter_id = cv.counter_id) left join process p "
+      + "on (cd.ref = p.upid) group by cd.counter_id";
```

Comment:
You are not actually using the pid, so undo this change.

Reply:
Done.